spec/requests/api/v1/posters_request_spec.rb

```diff
@@ -190,5 +190,73 @@
     posters = JSON.parse(response.body, symbolize_names: true)
     expect(posters[:meta][:count]).to eq(posters[:data].count)
   end
+
+  it "sorts results ascending by query parameters" do
+    poster_1 = Poster.create(
+      name: "REGRET",
+      description: "Hard work rarely pays off.",
+      price: 89.00,
+      year: 2018,
+      vintage: true,
+      img_url:  "https://plus.unsplash.com/premium_photo-1661293818249-fddbddf07a5d"
+    )
+    poster_2 = Poster.create(
+      name: "UNHAPPY",
+      description: "Hard work rarely pays off.",
+      price: 400.00,
+      year: 1738,
+      vintage: true,
+      img_url:  "https://plus.unsplash.com/premium_photo-1661293818249-fddbddf07a5d"
+    )
+    poster_3 = Poster.create(
+      name: "BIG SAD",
+      description: "Why smile when you can frown",
+      price: 120.00,
+      year: 2020,
+      vintage: false,
+      img_url:  "https://plus.unsplash.com/premium_photo-1661293818249-fddbddf07a5d"
+    )
+
+    get "/api/v1/posters?sort=asc"
+
+    expect(response).to be_successful
+    posters = JSON.parse(response.body, symbolize_names: true)
+    expect((posters[:data][0][:id]).to_s).to eq(poster_1[:id].to_s)
+    expect((posters[:data][2][:id]).to_s).to eq(poster_3[:id].to_s)
+  end
+
+  it "sorts results descending by query parameters" do
```

Comment:
it block will check the order of posters in response body by comparing response's first & last posters's ids data set order to first & last poster's ids. the order should be the opposite for descending ordering.